daisi/src/cpps/model/agv_fleet.cpp

```diff
@@ -57,7 +57,10 @@ mrta::model::Ability AGVFleet::getClosestExistingAbility(
     bool is_min = true;
 
     for (auto const &s : S) {
-      bool valid = !lessOrEqualAbility(s, m) || lessOrEqualAbility(m, s);
+      bool a = s <= m;
+      bool b = m <= s;
```

Comment:
imho, splitting conditions is good, but then the variables should be named appropriately based on what is checked.